lib/api/v3/activities/activity_representer.rb

```diff
@@ -27,36 +27,16 @@
 # See doc/COPYRIGHT.rdoc for more details.
 #++
 
-require 'roar/decorator'
-require 'roar/json/hal'
-
 API::V3::Utilities::DateTimeFormatter
 
 module API
   module V3
     module Activities
-      class ActivityRepresenter < Roar::Decorator
-        include Roar::JSON::HAL
-        include Roar::Hypermedia
-        include API::V3::Utilities::PathHelper
+      class ActivityRepresenter < ::API::Decorators::Single
         include API::V3::Utilities
 
-        self.as_strategy = API::Utilities::CamelCasingStrategy.new
-
-        def initialize(model, options = {})
-          @current_user = options[:current_user]
-
-          super(model)
-        end
-
-        property :_type, exec_context: :decorator
-
-        link :self do
-          {
-            href: api_v3_paths.activity(represented.id),
-            title: "#{represented.id}"
-          }
-        end
+        self_link path: 'activity',
```

Comment:
This is technically not wrong, it is working after all.

But usually I would use the symbol `:activity` here. I can't even tell you why I would do that. But I could bet that others would agree.